test/PolyhedralGeometry/subdivision_of_points.jl

```diff
@@ -14,8 +14,8 @@
     SubdivisionOfPoints
 
   @testset "alternative inputs" begin
-    @test collect(maximal_cells(square_by_incidence)) ==
-      collect(maximal_cells(square_by_weights))
+    @test issetequal(collect(maximal_cells(square_by_incidence)),
+      collect(maximal_cells(square_by_weights)))
```

Comment:
@benlorenz just fyi I patched another one (or the same, cannot remember) of these flaky tests, as it failed here